storage/flux/reader.go

```diff
@@ -54,6 +57,31 @@ type storeReader struct {
 	s storage.Store
 }
 
+type qWAC struct{}
```

Comment:
This was just added to return appropriate capabilities.

Reply:
Why do we need this duplicated capability struct? It is exactly the same as `readsWAC`.

To take one step back, I think we do not need to have the capability system in OSS at all since it does not have separate queryd and storaged services and operate everything in single node.